openstack/compute/v2/flavors/requests.go

```diff
@@ -21,22 +21,12 @@ type ListOptsBuilder interface {
 //
 // If no IsPublic argument is supplied in the request, OpenStack treats the FlavorType as Project.
 type FlavorType string
+
 const (
-        Project FlavorType =  "true"
-        Private FlavorType = "false"
-        All FlavorType = "None"
+	Project FlavorType = "true"
```

Comment:
I've reconsidered my earlier stance on the term "Access" in this context, because we're not directly talking about access.  The `All` type, for instance, means that all flavors are returned, not that, as I think it would imply, anyone can access that API call.  In fact only admins can use the `All` option.

I wonder if we should use the vernacular of the Nova API itself. I was trying to avoid it because I think it's so confusing--it's a ternary masquerading as a binary, and beyond that the request-response relationships are bizarre.  Reluctantly, I'm thinking of this:

    type IsPublic string

    const (
        Project IsPublic = "true"
        Private IsPublic = "false"
        All     IsPublic = "None"
    )

And used like:

    listOpts := flavors.listOpts{
       IsPublic: flavors.All,
    }